### DISCUSSION TITLE
[Blog] Quantum Music a Berlino [Math is in the air]

### DISCUSSION THREAD

**Discussion by medusamedusa** [Quantumland-art/community#21] [Bookshelf]
Hello! I wrote a post about the symposium, for a blog of applied mathematics. It is in Italian; the English version may appear the next month, hopefully before. Enjoy! :)

https://www.mathisintheair.org/wp/2023/10/quantum-music-a-berlino/

**Comment by omarcostahamido** [comment]
congrats, and thanks for sharing!
